docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,240 @@
+====
+
+**Currently, we require only https://grafana.com/grafana/plugins/marcusolsson-json-datasource/[JSON API], which has associated https://grafana.com/grafana/plugins/marcusolsson-json-datasource/?tab=installation[installation instructions].**
+
+However, if you:
+
+- Configure Grafana (v7.1+) dashboards through the use of provisioning files: https://grafana.com/docs/grafana/latest/administration/provisioning/#plugins[specify the plugins to install in your provisioning configuration file]. 
+- Are running Grafana in a Docker container: simply pass through an additional environment variable `GF_INSTALL_PLUGINS=$PLUGIN_NAME $VERSION` where the `$PLUGIN_NAME` and latest `$VERSION` can be found on the plugin's homepage (linked under "installation instructions" for each). 
+
+=== Grafana: Dashboards
+
+The CMOS dashboards can be found in the official GitHub repository under https://github.com/couchbaselabs/observability/tree/main/microlith/grafana/provisioning/dashboards[microlith/grafana/provisioning/dashboards]/.
+Download them and copy them to a folder accessible by your Grafana installation. 
+
+The relevant setting in the Grafana configuration file, typically named `grafana.ini`, is:
+
+[source, console]
+----
+[dashboards]
+default_home_dashboard_path = <path to dashboards>/couchbase-inventory.json
+----
+
+If you are using https://grafana.com/docs/grafana/latest/administration/provisioning/[Grafana provisioning], you will need to update the `providers.options.path` argument in your provisioning configuration file (typically named `grafana.yml`) instead.
+For example:
+
+[source, console]
+----
+providers:
+    - name: dashboards
+      type: file
+      ...
+      options:
+          path: /etc/grafana/provisioning/dashboards/
+      ...
+----
+
+=== Grafana: Data Sources
+
+Now we have the required plugins and dashboards installed, we need to configure Data Sources.
+
+. **Cluster Monitor (JSON API)**, via the Web UI: click on _Add Data Source_ and select _JSON API_. 
+  - The URL should point to the Cluster Monitor with a sub-path of `/api/v1`. The port should be `:7196`.
+  - Enable `basicAuth`, with the user and password you configured.
+. **Prometheus** - this should already be configured as a Data Source.
+. **Alertmanager** - this should already be configured as a Data Source. 
+  However, some of the dashboards query the Alertmanager API and so you will need to add a new Data Source in the same way as you did for the Cluster Monitor.
+  - This should be named _Alertmanager API_, also of type _JSON API_, and the URL should point to your Alertmanager instance. 
+  - Configure any authentication as needed.
+
+If you are utilizing Grafana provisioning for dashboards, and would instead like to specify this in a configuration file, then your `grafana.yml` should look something like this:
+[source, console]  
```

Comment:
```suggestion

[IMPORTANT]
====
The names of these two data sources must match the below snippet exactly, otherwise the dashboards may fail to provision.
====

[source, console]  
```